pnpm-workspace.yaml

```diff
@@ -1,9 +1,13 @@
 packages:
   - "examples/*"
+  - "examples/*/*"
+  - "examples/*/*/*"
   - "packages/*"
   - "integration-tests"
+  - "integration-tests/test-api"
```

Comment:
We don't want to include the test-api here, as it is not expected to have it's dependencies installed via pnpm.

Reply:
I didn't document it, but the test api should really just be ran via docker.

Reply:
removed it!
I found it handle to be able to run it on my machine instead of docker for quicker debugging but I think leaving it to only be run on docker is fine!